.github/workflows/ci.yml

```
@@ -11,15 +11,24 @@ jobs:
-        uses: astral-sh/setup-uv@v3
-
       - name: Set up Python
-        run: uv python install 3.11
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.11'
 
-      - name: Install the dependencies
-        run: uv sync 
+      - name: Create and activate virtual environment
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:2:53: Double quote to prevent globbing and word splitting

(shellcheck)